substrate/primitives/weights/src/lib.rs

```diff
@@ -52,28 +50,6 @@ pub mod constants {
 	pub const WEIGHT_PROOF_SIZE_PER_KB: u64 = 1024;
 }
 
-/// The old weight type.
-///
-/// NOTE: This type exists purely for compatibility purposes! Use [`weight_v2::Weight`] in all other
-/// cases.
-#[derive(
-	Decode,
-	Encode,
-	CompactAs,
-	PartialEq,
-	Eq,
-	Clone,
-	Copy,
-	RuntimeDebug,
-	Default,
-	MaxEncodedLen,
-	TypeInfo,
-)]
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-#[cfg_attr(feature = "serde", serde(transparent))]
-#[deprecated(note = "Will be removed soon; use `Weight` instead.")]
-pub struct OldWeight(pub u64);
```

Comment:
Yea i think that is pretty much it.

It is meant as a beginner issue, so maybe it too easy for you 😆